open-reac/src/main/java/com/powsybl/openreac/parameters/input/VoltageLevelLimitsOverrideInput.java

```diff
@@ -32,62 +32,70 @@ public class VoltageLevelLimitsOverrideInput implements AmplInputFile {
 
     private final Map<String, Pair<Double, Double>> normalizedVoltageLimitsOverride;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(VoltageLevelLimitsOverrideInput.class);
     private static final String OVERRIDE_ON_VOLTAGE_LEVEL = "Override on voltage level ";
+    private static final double VOLTAGE_LIMIT_LOW_THRESHOLD = 0.85;
+    private static final double VOLTAGE_LIMIT_HIGH_THRESHOLD = 1.15;
+    private static final double VOLTAGE_LIMIT_TOLERANCE = 5;
 
-    public VoltageLevelLimitsOverrideInput(List<VoltageLimitOverride> voltageLimitsOverrides, Network network) {
+    public VoltageLevelLimitsOverrideInput(List<VoltageLimitOverride> voltageLimitsOverrides, Network network, ReportNode reportNode) {
```

Comment:
The class is public, so it could be used by another project (even if it seems very unlikely).
Could you keep a constructor with the previous signature, or report this breaking change?

To keep the previous signature:
```suggestion
    public VoltageLevelLimitsOverrideInput(List<VoltageLimitOverride> voltageLimitsOverrides, Network network) {
        this(voltageLimitsOverrides, network, ReportNode.NO_OP);
    }

    public VoltageLevelLimitsOverrideInput(List<VoltageLimitOverride> voltageLimitsOverrides, Network network, ReportNode reportNode) {
```